.github/workflows/repl.yml

```diff
@@ -20,4 +20,4 @@ jobs:
           github-token: ${{ secrets.BOT_TOKEN }}
           script: |
             const { default: repl } = await import('${{ github.workspace }}/scripts/actions/repl.js');
-            await repl();
+            await repl(github, context);
```

Comment:
These are not JS variables -- could you move the action to https://github.com/babel/actions and follow how, for example, https://github.com/babel/actions/blob/89fc1d36b649d3b6f2ba40e1051555c999d69842/publish-github-release/main.js#L5 gets these values?

Reply:
They are defined by `actions/github-script`, but if you feel better moving to https://github.com/babel/actions I can do that too.